sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -377,6 +378,36 @@ public CosmosClientBuilder contentResponseOnWriteEnabled(boolean contentResponse
         return this;
     }
 
+    /**
+     * Gets the boolean which indicates whether bulk is enabled for the requests or not.
+     *
+     * Allows optimistic batching of requests to service. Setting this option might impact the latency of the operations.
+     * Hence this option is recommended for non-latency sensitive scenarios only.
+     *
+     * By-default, this is false.
+     *
+     * @return a boolean indicating whether bulk is enabled for the requests or not.
+     */
+    boolean isBulkExecutionEnabled() {
+        return bulkExecutionEnabled;
+    }
+
+    /**
+     * Sets the boolean to enable bulk for the requests.
+     *
+     * Allows optimistic batching of requests to service. Setting this option might impact the latency of the operations.
+     * Hence this option is recommended for non-latency sensitive scenarios only.
+     *
+     * By-default, this is false.
+     *
+     * @param bulkExecutionEnabled a boolean indicating whether to enable bulk for the requests
+     * @return current cosmosClientBuilder
+     */
+    public CosmosClientBuilder bulkExecutionEnabled(boolean bulkExecutionEnabled) {
```

Comment:
naming - should be enableBulkExecution instead?

Reply:
naming is correct. `bulkExecutionEnabled` is consistent with java naming.